.github/workflows/gh-pages.yml

```diff
@@ -4,24 +4,24 @@ on:
   workflow_dispatch:
   push:
     tags:
-    - '*'
+      - '*'
```

Comment:
this is why we don't like yaml

Reply:
TBF I think it worked regardless, it just didn't match the rest of the file (nor my expectations)

Reply:
![](https://newfastuff.com/wp-content/uploads/2019/07/IJELjdB.png)

Reply:
😂